modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,385 @@
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+// TODO: downloads available?
+Failed Requests Volume by Response Code::
+Total number of unsuccessful requests at specific time points, grouped by the response code (such as *2xx*)
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+// TODO: paths -> Paths and "N" to "20" if that's the limit (SATHISH)
+Failed Requests by Top N Paths::
+Total number of unsuccessful requests at specific time points from the 20 endpoints with the most requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//end::api-detail-errors[]
+//
+//
+//-------------------------------------------------------------//
+// API DETAILS: Performance tab
+//----------------------------
+//tag::api-detail-performance[]
+
+//TODO: issue: RESPONSE TIME UNIT MISSING from chart (SATHISH)
+//
+//can download as CSV, XLS
+Top 20 Paths by Average Response Time::
+Average response time of the 20 most requested endpoints 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: 95p (to standardize) or capitalize "percentile" -> "Percentile" (SATHISH)
+//TODO: issue: missing time units on X axis
+//TODO: issue: duplication of time values along the X axis
+//You can download as CSV, XLS
+Top 20 Paths by 95th Percentile Response Time::
+95th percentile value for each of the 20 most requested endpoints
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::api-detail-performance[]
+//-------------------------------------------------------------//
+//
+//-------------------------------------------------------------//
+// API DETAILS: Client Applications tab
+//-------------------------
+//tag::api-detail-client-apps[]
+Request Volume by Location::
+Maps with the number of requests to geographic areas across the globe
++
+You can click the request volume number on the map to get more detailed information.
+//end::api-detail-client-apps[]
+//-------------------------------------------------------------//
+
+
+//========================//
+//SHARED in multiple lists (Apps and APIs):
+//========================//
+//
+// shared chart: Overview tab
+//tag::average-latency[]
+Average Latency::
+Average latency, in milliseconds (ms), for requests to this entity
+//end::average-latency[]
+//
+// shared chart: Overview tab
+//tag::p95-latency[]
+P95 Latency::
+Latency value, in milliseconds (ms), for 95th percentile (p95) request for this entity
+//end::p95-latency[]
+//
+//  shared chart: Overview tab
+//tag::total-requests[]
+//TODO: Requests -> Request" (SATHISH)
+Total Requests Volume::
+Total number of requests to this entity
+//end::total-requests[]
+//
+//  shared chart: Overview tab
+//tag::request-latency[]
+Request Latency::
+Latency, in milliseconds (ms), at specific time points for the average request and the p90, p95, and p99 percentile requests
```

Comment:
Global: do you need "points"? Can you say "specific times"?

Reply:
These are on graphs, so I want to emphasize that the numbers are for specific points on the x axis, and those points are represented as times.